spec/expected/connection/chroot-systemd-pid.txt

```diff
@@ -7,7 +7,8 @@ Wants=network-online.target
 
 [Service]
 Type=simple
-ExecStartPre=-/usr/bin/pkill -f "stunnel /etc/stunnel/stunnel.conf" -F /var/stunnel/var/opt/run/stunnel.pid ; rm /var/stunnel/var/opt/run/stunnel.pid
+ExecStartPre=/bin/bash -c 'if test -f /var/stunnel/var/opt/run/stunnel.pid; then /usr/bin/pkill -f "stunnel /etc/stunnel/stunnel.conf" -F /var/stunnel/var/opt/run/stunnel.pid; fi'
```

Comment:
The dash right after the equals here means that this step can have any exit code, if that makes this logic easier

Reply:
I opted to not use the dash to allow fail option, because systemctl status stunnel always prints out error messages about not being able to pkill or remove the lock file.

Reply:
Oh, neat.